docs/variants/novacustom_nv4x/post_install.md

```diff
@@ -26,26 +26,43 @@ enable the touchpad hotkey to work under Linux, follow the steps below:
 After executing these steps, it should be possible to enable and disable the
 touchpad using the touchpad hotkey (Fn+F1) on the keyboard when using GNOME.
 
+## Nvidia drivers (Ubuntu 22.04)
+
+For proper functioning of the sleep mode on Ubuntu 22.04, it is required to
```

Comment:
Maybe the word `working` should be better than `functioning`?

Reply:
replaced